Ruveyda

Ruveydabal opened the pull request:
I have done all 3 of the exercises.
I also made a README file for all 3 of them, but I am not sure if I did that right. 
Can you check if I did the exercises and the README file correct?

Ruveydabal committed:
Add .gitattributes and .gitignore.

Ruveydabal committed:
Add project files.

Ruveydabal committed:
code

Ruveydabal committed:
test

Ruveydabal committed:
test

adev3141 committed:
Merge branch 'main' into Ruveyda

Ruveydabal committed:
Merge branch 'Ruveyda' of https://github.com/adev3141/GIMInterns into Ruveyda

Ruveydabal committed:
test

testing if it worked

Ruveydabal committed:
Merge branch 'main' into Ruveyda

Ruveydabal committed:
merge

merge with main branch

Ruveydabal committed:
merge

Ruveydabal committed:
delete

did not need it have a copy somewhere else

Ruveydabal committed:
new project

start excercise c#

Ruveydabal committed:
not sure

not sure what happend here :)

Ruveydabal committed:
user

class user start

Ruveydabal committed:
main

start in main class

Ruveydabal committed:
Library

start in Library

Ruveydabal committed:
Book

Start in Book class

Ruveydabal committed:
Library

 the methods for Library

Ruveydabal committed:
user

small change

Ruveydabal committed:
main

tested if my methods work

Ruveydabal committed:
Book

changed it  a bit

Ruveydabal committed:
PremiumUser

made a start in PremiumUser

Ruveydabal committed:
user

changed namespace

Ruveydabal committed:
Book

changed it again

Ruveydabal committed:
Library

changed namespace and other smaller things

Ruveydabal committed:
Main

changed namespace

Ruveydabal committed:
Library

added virtual to method BorrowBook

Ruveydabal committed:
User

Made RequestBorrowBook don't know if I will keep using it or not

Ruveydabal committed:
Main

just testing the methods that I changed/made

Ruveydabal committed:
PremiumUser

Added namespace and class PremiumUser. it is far from done

Ruveydabal committed:
User

changed commentary

Ruveydabal committed:
Library

Changed method ReturnBook and method BorrowBook

Ruveydabal committed:
Main

Testing of the methods are working like I inteded them to

Ruveydabal committed:
PremiumUser

Changed commentary

Ruveydabal committed:
Main

Testing if the methods work like I inteded

Ruveydabal committed:
PremiumUser

RequestBorrowBook works now

Ruveydabal committed:
User

RequestBorrowBook works now

Ruveydabal committed:
PremiumUser

Added a method

Ruveydabal committed:
User

Added a method

Ruveydabal committed:
Shellbuddy

Installed shellbuddy

Ruveydabal committed:
PemiumUser

PremmiumUser can now borrow more than one book

Ruveydabal committed:
IUser

Made Interface User

Ruveydabal committed:
Library

Made a method that can find a book by the title

Ruveydabal committed:
Main

Tested a method that I made

Ruveydabal committed:
folder

Put them in a folder with the name Library Management System

Ruveydabal committed:
folder

Added them in the folder

Ruveydabal committed:
Student Record System

Started on Student Record Sytem exercise

Ruveydabal committed:
Place

Changed there place in my filess

Ruveydabal committed:
Student management

Added them to a new file

Ruveydabal committed:
new file

Added them to a new fille place

Ruveydabal committed:
Student management system

Changed added AddStudents in Student

Ruveydabal committed:
Student

Put a comment in

Ruveydabal committed:
Student

Changed comment

Ruveydabal committed:
Main

Tested if my method works how I wanted to

Ruveydabal committed:
Student

Made method AddStudent and LookUpStudent

Ruveydabal committed:
Create HelpDesk

Ruveydabal committed:
Main

Tested if it worked like I meant it to work

Ruveydabal committed:
Student

Changed Student

Ruveydabal committed:
Create StudentManagement

Ruveydabal committed:
StudentManagemeng

Changed some things

Ruveydabal committed:
Student

Removed LookUpStudent from Student

Ruveydabal committed:
Main

Tested if it worked

Ruveydabal committed:
Main

Testing

Ruveydabal committed:
Helpdesk

Deleted things that i did not need

Ruveydabal committed:
Main

Testing

Ruveydabal committed:
Helpdesk

Changed AddToQueue so that it also shows the queue at the end.

Ruveydabal committed:
Student

Added AverageGrade method

Ruveydabal committed:
HelpDesk

Changed comments

Ruveydabal committed:
Main LibraryMangementSytem

Made it so you can run it

Ruveydabal committed:
Create Main StudentFileManagement

Ruveydabal committed:
Create

Ruveydabal committed:
Create student for file system

Ruveydabal committed:
Main StudentRecordManagement

made it so you can run it

Ruveydabal committed:
Update Student

record management

Ruveydabal committed:
StudentManagement

Changed somethings

Ruveydabal committed:
update

Ruveydabal committed:
Update Main file management

Ruveydabal committed:
Create Serialization

Made methods

Ruveydabal committed:
Student file management

Changed it up a bit

Ruveydabal committed:
Removed them into a folder

Ruveydabal committed:
Removed into folder

Ruveydabal committed:
Removed into folder

Ruveydabal committed:
Removed into folder

Ruveydabal committed:
Added into folder

Ruveydabal committed:
Added into folder

Ruveydabal committed:
Added into folder

Ruveydabal committed:
Deleted

Did not use them

Ruveydabal committed:
StudentManagement

Ruveydabal committed:
Main Library

Made it so that it stops running if you choose 6

Ruveydabal committed:
Updat main Student file

Ruveydabal committed:
Managestudent

Created ManageStudent class

Ruveydabal committed:
Main Student record

Made it so it stops running if you choose 9

Ruveydabal committed:
Deleted StudentManagement

Made a copy of it

Ruveydabal committed:
Changed file

Changed the name of the file to Library Management

Ruveydabal committed:
changed files

Changed files again so that the work again

Ruveydabal committed:
all works now

I just need to make it so that when a user is not a primium user they cannot borrow more than one book.

Ruveydabal committed:
User or Premium user

It now asks if you are a premium user or not.

Ruveydabal committed:
Create README.md

Created a readme file. not sure if i did it right

Ruveydabal committed:
Create README.md

Made a readme file for se second assignment. I don't know if i made it the right way

Ruveydabal committed:
Create README.md

Created a readme file for Student file management, I'm not sure if i did it correctly

adev3141 commented on StageGIM/StageGIM/Assignment-1/Book.cs:
```diff
@@ -0,0 +1,14 @@
+namespace LibraryManagementSystem
+{
+    public class Book(string title, string author, string isbn, string genre, bool isAvailable)
+    {       //is this oke to be public?
```
yes it is

adev3141 commented on StageGIM/StageGIM/Assignment-1/IUser.cs:
```diff
@@ -0,0 +1,15 @@
+    {//not sure if I did this right
+        void BorrowBook();
+
+        void RetunrBook();
```
please fix the spellings

adev3141 commented on StageGIM/Student Management/Student Management/Assignment-2/Main.cs:
```diff
@@ -0,0 +1,68 @@
+
+                string Input =Console.ReadLine();
+
+                switch (Input)
```
mention the cases in comments above each case below

Ruveydabal replied:
Did I do it like you meant what I needed to do?

Ruveydabal committed:
update IUser

Fixed the spelling

adev3141 commented on StageGIM/Student Management/Student Management/Assignment-2/Student.cs:
```diff
@@ -0,0 +1,71 @@
+
+        public List<int> Grade = new List<int>();
+
+        public List<Student> StudentList = new List<Student>();
```
why does student have a student list? a class has a student list, student doesn't

Ruveydabal replied:
I added a new class named ClassRoom and put the student list there.

Ruveydabal committed:
Readme

testing if the readme worked

Ruveydabal committed:
Merge branch 'Ruveyda' of https://github.com/adev3141/GIMInterns into Ruveyda

Ruveydabal committed:
Update Main

Added comments above each case

Ruveydabal committed:
Created ClassRoom class

Created ClassRoom class so I can put the student list there and not in the student class.

Ruveydabal committed:
changed everywhere where StudentList is used

changed everywhere where StudentList is used so that it comes from ClassRoom and not from student

Ruveydabal committed:
created file

added GML file to folder for easy acces

Ruveydabal committed:
editorconfig

not sure what this does

Ruveydabal committed:
created new solution

made a new solution where i can work

Ruveydabal committed:
created ConvertClass

class where it converts the files

Ruveydabal committed:
created GmlCoordinateprivate class

it is a private class that is going to have duplicate code from the GmlCoordinate class

Ruveydabal committed:
created GmlCoordinate class

created GmlCoordinate Class here it is going to get the coordinates from the lines and points

Ruveydabal committed:
created Main class

changed main to Main this is going to be where the program is going to run

Ruveydabal committed:
created main

this is when it was named main but now  i changed it so it is named Main

Ruveydabal committed:
Converter

It shows coordinates from an gml file (hardcoded).  Shows both point and lines but you can't choose one yet.

Ruveydabal committed:
Converter

You can now choose if you want to see the points or the lines

Ruveydabal committed:
Main

You can now add your own file path to find the gml file that you want to see.

Ruveydabal committed:
Comments

Changed some of the comments in the code
